google/auth/_helpers.py

```diff
@@ -230,3 +232,35 @@ def unpadded_urlsafe_b64encode(value):
         Union[str|bytes]: The encoded value
     """
     return base64.urlsafe_b64encode(value).rstrip(b"=")
+
+
+def get_user_password(text):
+    """Get password from user.
+
+    Override this function with a different logic if you are using this library
```

Comment:
Are users expected to override this function? (If so, it seems odd that it is in the private `_helpers.py`)

Reply:
Moved the two methods to where they are used, i.e., `get_user_password` to `challenges.py`, and `is_interactive` to `reauth.py`.